sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidCertainMethodCheck.java

```diff
@@ -173,6 +173,20 @@ public void visitToken(DetailAST ast) {
         }
     }
 
+    /**
+     * Count the parameters given to a method call.
+     * @param expressionList The expression list containing the parameters.
+     * @return The number of parameters.
+     */
+    private int getMethodCallParameterCount(DetailAST expressionList) {
+        int paramCount = 0;
+        if (expressionList.getChildCount() > 0) {
+            // At least one parameter, count commas
```

Comment:
this comment is misleading.
please make it as "amount of parameters is always +1 of amount of commas" or remove this comment.